src/components/settings/SettingsSSOTab/NewExistingSSOConfigs.jsx

```diff
@@ -0,0 +1,196 @@
+import {
+  CardGrid,
+  Skeleton,
+  useToggle,
+} from '@edx/paragon';
+import { useQuery, useQueryClient } from '@tanstack/react-query';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import LmsApiService from '../../../data/services/LmsApiService';
+import NewSSOConfigAlerts from './NewSSOConfigAlerts';
+import NewSSOConfigCard from './NewSSOConfigCard';
+
+const FRESH_CONFIG_POLLING_INTERVAL = 30000;
+const UPDATED_CONFIG_POLLING_INTERVAL = 2000;
+
+const NewExistingSSOConfigs = ({
+  configs, refreshBool, setRefreshBool, enterpriseId,
+}) => {
+  const [inactiveConfigs, setInactiveConfigs] = useState([]);
+  const [activeConfigs, setActiveConfigs] = useState([]);
+  const [inProgressConfigs, setInProgressConfigs] = useState([]);
+  const [untestedConfigs, setUntestedConfigs] = useState([]);
+  const [liveConfigs, setLiveConfigs] = useState([]);
+  const [notConfiguredConfigs, setNotConfiguredConfigs] = useState([]);
+  const [queryForTestedConfigs, setQueryForTestedConfigs] = useState(false);
+  const [queryForConfiguredConfigs, setQueryForConfiguredConfigs] = useState(false);
+  const [intervalMs, setIntervalMs] = React.useState(FRESH_CONFIG_POLLING_INTERVAL);
+  const [loading, setLoading] = useState(false);
+  const [showAlerts, openAlerts, closeAlerts] = useToggle(false);
+
+  const queryClient = useQueryClient();
+
+  useQuery({
+    queryKey: ['todos'],
+    queryFn: async () => {
+      const res = await LmsApiService.listEnterpriseSsoOrchestrationRecords(enterpriseId);
+      const inProgress = res.data.filter(
+        config => (config.submitted_at && !config.configured_at) || (config.configured_at < config.submitted_at),
+      );
+      const untested = res.data.filter(config => !config.validated_at || config.validated_at < config.configured_at);
+
+      if (queryForConfiguredConfigs) {
+        if (inProgress.length === 0) {
+          setRefreshBool(!refreshBool);
+          setQueryForConfiguredConfigs(false);
+        }
+      }
+
+      if (queryForTestedConfigs) {
+        if (untested.length === 0) {
+          setRefreshBool(!refreshBool);
+          setQueryForTestedConfigs(false);
+        }
+      }
+
+      if (inProgress.length === 0 && untested.length === 0) {
+        queryClient.invalidateQueries({ queryKey: ['todos'] });
+      }
+
+      return res.data;
+    },
+    // Refetch the data every second
+    refetchInterval: intervalMs,
+    enabled: queryForTestedConfigs || queryForConfiguredConfigs,
+    refetchOnWindowFocus: true,
+  });
+
+  useEffect(() => {
+    const inactive = configs.filter(config => config.active === false);
+    const active = configs.filter(config => config.active === true);
```

Comment:
nit: Could simplify a bit using [lodash.partition](https://lodash.com/docs/4.17.15#partition)
```suggestion
    const [active, inactive] = _.partition(configs, config => config.active);
```